docs/user/ai_model_serving.md

```diff
@@ -0,0 +1,482 @@
+Keep in mind that in MicroShift they are created directly as ServingRuntimes in `redhat-ods-applications`, not as templates,
+so they need to be copied to workload's namespace.
+
+## General usage overview
+
+1. Prepare model for serving
+1. Configure OS and MicroShift for the hardware - driver & device plugin
+1. Install AI Model Serving for MicroShift
+1. (Optional) Package model into an OCI image (ModelCar)
+1. Select suitable Serving Runtime (Model Server)
+1. Copy ServingRuntime Custom Resource from `redhat-ods-applications` to your own namespace
+1. Create InferenceService
+1. Create Route
+1. Make requests against model server
+
+## Setting up hardware - drivers and device plugins
+
+In order to enable GPU/hardware accelerator for Red Hat Device Edge,
+you must follow partner's guidance on installing Operator, or driver for RHEL and Device Plugin for Kubernetes.
+Operators may be more convenient, but installing only the driver and device plugin may be more resource efficient.
+
+MicroShift does not provide support for partner's procedure.
+In case of any problems consult partner's documentation or support.
+Following links serve only as examples / pointers and may not include everything needed.
+- NVIDIA:
+  - [GPU Operator](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/index.html)
+  - [Device Plugin](https://github.com/NVIDIA/k8s-device-plugin)
+  - [Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html)
+  - [Driver](https://docs.nvidia.com/datacenter/tesla/driver-installation-guide/index.html) / [CUDA Toolkit](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/)
+- Intel Guadi
+  - [Base Operator](https://docs.habana.ai/en/latest/Installation_Guide/Additional_Installation/Kubernetes_Installation/Kubernetes_Operator.html)
+  - [Device Plugin](https://docs.habana.ai/en/latest/Installation_Guide/Additional_Installation/Kubernetes_Installation/Intel_Gaudi_Kubernetes_Device_Plugin.html)
+  - [Driver](https://docs.habana.ai/en/latest/Installation_Guide/Driver_Installation.html)
+- AMD
+  - [Operator](https://instinct.docs.amd.com/projects/gpu-operator/en/latest/)
+  - [Device Plugin](https://github.com/ROCm/k8s-device-plugin)
+  - [Driver](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/install/install-overview.html)
+
+## Step by step guide
+
+Below is an example usage of AI Model Serving for MicroShift.
+It uses OpenVino Model Server (OVMS) and resnet50 model.
+OVMS can run on CPU, no hardware accelerator is needed for this example.
+
+### Installing AI Model Serving for MicroShift
+
+`microshift-ai-model-serving` RPM contains manifests that deploy `kserve` and `ServingRuntimes` in `redhat-ods-applications` namespace.
+
+To install AI Model Serving for MicroShift run following command:
+```bash
+sudo dnf install microshift-ai-model-serving
```

Comment:
Should we mention here to install both RPMs?
 - `microshift-ai-model-serving`
 - `microshift-ai-model-serving-release-info`